README.md

```diff
@@ -70,3 +70,22 @@ Permit user2@email.com to bq-project-id access as READER
 ...
 
 ```
+
+## Completion
+Completion is available for bash or zsh.
+Download projects, datasets, users list data via GCP API.
+
+### bash
+```
+bqiam completion bash > /path/to/bash-completion/completions/bqiam
+```
+
+### zsh
+```
+bqiam completion bash > /path/to/zsh-completions/_bqiam
```

Comment:
`bqiam completion zsh` ?

Reply:
@kitagry I fixed it on the PR below! Please check
https://github.com/hirosassa/bqiam/pull/39

Reply:
:+1: